apps/llm-client/src/components/chat/message-bubble.test.tsx

```
@@ -0,0 +1,41 @@
+import { describe, expect, it } from "vitest";
+import { render, screen } from "@testing-library/react";
+import { MessageBubble } from "./message-bubble";
+
+describe("MessageBubble", () => {
+  it("renders 'User:' label for user messages", () => {
+    render(<MessageBubble role="user" content="hi" />);
```

Biome (2.4.10)

[error] Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.

(lint/a11y/useValidAriaRole)